layrry-config-toml/src/main/java/org/moditect/layrry/config/TomlLayersConfigParser.java

```diff
@@ -13,63 +13,34 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.moditect.layrry.internal.descriptor;
+package org.moditect.layrry.config;
```

Comment:
I'd suggest `org.moditect.layrry.config.toml`, so to avoid the split package with YAML module. Or better yet, `org.moditect.layrry.config.toml.internal`, as I don't think this is user-facing?

Reply:
Ah yes, split packages. Will take care of that.